src/Resource/GetSpecificRate.php

```diff
@@ -30,7 +32,7 @@ public function getProviderClassName(): string
 
     public function fetch(ImportConnector $connector): \Iterator
     {
-        $response = \json_decode(
+        $response[] = \json_decode(
```

Comment:
This is wrapping your record in an outer array, which is not correct. This should just be `$response` instead of `$response[]`.